src/Symfony/Bundle/TwigBundle/composer.json

```diff
@@ -21,7 +21,7 @@
         "symfony/twig-bridge": "^3.2.1",
         "symfony/http-foundation": "~2.8|~3.0",
         "symfony/http-kernel": "~2.8.16|~3.1.9|^3.2.2",
-        "twig/twig": "~1.28|~2.0"
+        "twig/twig": "~2.2"
```

Comment:
That's not possible. We cannot drop support for Twig 1.x. But the good news is that we are still maintaining Twig 1.x and 2.x. So the constraint can be `^1.29|^2.2`.